lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -370,6 +373,15 @@ def merchant_initiated_unscheduled(options)
         return 'Y' if options.dig(:stored_credential, :initiator) == 'merchant' && options.dig(:stored_credential, :reason_type) == 'unscheduled'
       end
 
+      def recurring_txn(options)
```

Comment:
We should spell out `transaction` here, just to be consistent with the rest of the code.